src/urbit/mar/chess/result.hoon

```diff
@@ -0,0 +1,19 @@
+/+  chess
+=,  format
```

Comment:
It's a non-issue, but feel free to add a `=,  chess` below this line if you like. You can't just do `=,  chess, format`, which is odd.